client/client.go

```diff
@@ -1108,6 +1370,41 @@ func (c *Client) runHandleEvent(msg *wamp.Event) {
 			msg.Subscription)
 		return
 	}
+
+	// Check and handle Payload PassThru Mode
+	// @see https://wamp-proto.org/wamp_latest_ietf.html#name-payload-passthru-mode
+	// Since it is async publication, and no link to
+	// original publication - as it was already published
+	// we can not reply with error, only log it.
+	// Although the router should handle it
+	if pptScheme, _ := msg.Details[wamp.OptPPTScheme].(string); pptScheme != "" {
+
+		if !isPPTSchemeValid(pptScheme) {
+			c.log.Print(ErrPPTSchemeInvalid.Error())
```

Comment:
this is only logged on the client side, so let's add some more context to make the logs actually useful:
```suggestion
			c.log.Print(fmt.Sprintf("cannot process even with invalid ppt schema %qt: %v", pptSchema, ErrPPTSchemeInvalid))
```